src/pages/Jbrowse.jsx

```diff
@@ -22,6 +22,7 @@ const App = (props) => {
 				storeClass : "JBrowse/Store/SeqFeature/BAM",
 				label : bam,
 				type : "JBrowse/View/Track/Alignments2",
+				chunkSizeLimit: 40000000
```

Comment:
is there a reasoning behind 40MB? just curious, not very familiar with the data

Reply:
Just rounding to the upper limit of the bug, I don't know how to find the chunkSizes for every BAM track. It's possible this issue may arise again with bigger files but for now this'll do!